staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructured.go

```diff
@@ -307,8 +307,8 @@ func (u *Unstructured) GetOwnerReferences() []metav1.OwnerReference {
 
 func (u *Unstructured) SetOwnerReferences(references []metav1.OwnerReference) {
 	var newReferences = make([]map[string]interface{}, 0, len(references))
-	for i := 0; i < len(references); i++ {
-		newReferences = append(newReferences, setOwnerReference(references[i]))
+	for _, reference := range references {
+		newReferences = append(newReferences, setOwnerReference(reference))
```

Comment:
Yeah, it's done the way it is for performance reasons. I think we can close this PR.